docs/elements/examples/primitives.mdx

```diff
@@ -0,0 +1,54 @@
+---
+description: Example implementations using Clerk Elements primitives.
+---
+
+# Primitives
+
+## OTP Input
+
+The following example demonstrates how to use the `otp` input type with the render prop to create a custom OTP input field for phone and email codes, that is animated with Framer Motion.
+
+<CodeBlockDemo demos={[
+  {
+    key: 'otp-input',
+    demo: '/demo/elements/sign-in/otp-input',
+    style: {
+      height: 500
+    }
+  }
+]}>
+```tsx {{ title: 'OTP Input', collapsible: true }}
+<Clerk.Input
+  type="otp"
+  required
+  className="flex justify-center gap-x-1"
```

Comment:
Personally I prefer less typing so `gap-1` for me here but not a blocker if you prefer explicitly using x and y with gap